packages/docprovider/src/yprovider.ts

```diff
@@ -190,6 +190,8 @@ export class WebSocketProviderWithLocks
    * Returns a Promise that resolves to the lock number.
    */
   acquireLock(): Promise<number> {
+    // keep two locks the first and the second
```

Comment:
Not sure about this comment. What do you mean?

Reply:
Ouch. Nothing some notes I wrote for me, and I forgot to remove them.